functions/snippets/concepts/src/main/java/com/example/functions/concepts/eventpojos/GcsEvent.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.functions.concepts.eventpojos;
+
+import java.util.Date;
+
+// [START functions_helloworld_gcs_event]
+public class GcsEvent {
```

Comment:
Do we expect users to copy and paste this class? Why are there multiple copies of this class in the samples rather than just one copy?

Reply:
Ideally we'd move them into a separate package in the central Maven repo - but copying is the only way to avoid "sibling dependencies" for now.